pkg/tests/testdata/local/Pulumi.yaml

```diff
@@ -0,0 +1,10 @@
+name: local
```

Comment:
Should test provider be listed as a plugin here? Does program test add the plugins to the project for us?

Reply:
Yeh program test adds it for us. We can't add it directly here because program test also copies the test code to a temp dir so a plain relative path doesn't work.